### PR TITLE
Enable Publishing to Ballerina Central on Release

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

**Review by ThisaruGuruge — DISMISSED** [review]

File: uuid-ballerina/build.gradle [hunk 1]
```diff
@@ -155,39 +175,24 @@ task ballerinaTest {
     }
 }
 
-test {
-    dependsOn(ballerinaTest)
-}
-
```
Comment (ThisaruGuruge) [inline uuid-ballerina/build.gradle:161]: Why are we removing this?

Comment (MadhukaHarith92) [reply uuid-ballerina/build.gradle]: Fixed in edd7771

File: .github/workflows/publish-release.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
               with:
                   java-version: 11
             - name: Set version env variable
-              run: echo "VERSION=$((grep -w 'version' | cut -d= -f2) < gradle.properties | cut -d- -f1)" >> $GITHUB_ENV
+                run: echo "VERSION=$((grep -w 'version' | cut -d= -f2) < gradle.properties | rev | cut --complement -d- -f1 | rev)" >> $GITHUB_ENV
```
Comment (aashikam) [inline .github/workflows/publish-release.yml:19]: Indentation issue?

**Review by ThisaruGuruge — DISMISSED** [review]

File: .github/workflows/publish-release.yml (hunk 2 above)
Comment (ThisaruGuruge) [reply .github/workflows/publish-release.yml:19]: yeah, we have to fix this. Nice catch.

Comment (MadhukaHarith92) [reply .github/workflows/publish-release.yml]: Fixed.

**Review by aashikam — APPROVED** [review]
LGTM

**Pull request merged**